web/packages/design/src/Link/Link.jsx

```diff
@@ -20,13 +20,7 @@ import styled from 'styled-components';
 
 import { color, space } from 'design/system';
 
-function Link({ ...props }) {
-  return <StyledButtonLink {...props} />;
-}
-
-Link.displayName = 'Link';
-
-const StyledButtonLink = styled.a.attrs({
+const Link = styled.a.attrs({
```

Comment:
This change can stay, `<StyledButtonLink>` was unnecessarily wrapped in `Link` instead of being exported directly.

Reply:
Welp, this caused the inferred type of `<Link>` change, which broke a couple of other places, most importantly https://github.com/gravitational/teleport/pull/51698#discussion_r1956013904.

I reverted the change and the type check seems to pass.